doc/manual/manual.tex

```diff
@@ -6949,9 +6949,124 @@ \subsubsection{The 2D cylindrical shell benchmarks by Davies et al.}
+center of the domain, causing dynamic topography at the surface. The details for the setup
+are in the caption of Figure~\ref{fig:crameri-benchmark-rising-blob}.
+
+\begin{figure}
+  \begin{center}
+    \includegraphics[width=0.95\textwidth]{cookbooks/benchmarks/crameri/rising_blob}
+  \end{center}
+  \caption{\it Setup for the dynamic topography benchmark. Again, the domain is $2800$ km 
+  wide and $700$ km high.  A $100$ km thick lid with viscosty $10^{23}$ overlies a mantle
+  with viscosity $10^{21}$.  Both the lid and the mantle have a density of $3300\,kg/m^3$.
+  A blob with diameter $100$ km lies $300$ km from the bottom of the domain.  The blob has 
+  a density of $3200 kg/m^3$ and a viscosity of $10^{20}$ Pa s.}
+  \label{fig:crameri-benchmark-rising-blob}
+\end{figure}
+
+Case two requires higher resolution and longer time integrations than case one. The benchmark
+is over 20 million years and builds dynamic topography of $\sim 800$ meters.  
+
+\begin{figure}
+  \begin{center}
+    \includegraphics[width=0.95\textwidth]{cookbooks/benchmarks/crameri/crameri_2_comparison}
+  \end{center}
+  \caption{\it Evolution of topography for the dynamic topography benchmark. The maximum topography
+   is shown as a function of time, for \aspect as well as for several other codes participating in
+   the benchmark. This benchmark shows considerably more scatter between the codes.}
+  \label{fig:crameri-2-comparison}
+\end{figure}
+
+Unlike the first benchmark, for case two there is no (semi) analytical solution to compare against.
+Furthermore, the time integration for this benchmark is much longer, allowing for errors to 
+accumulate. As such, there is considerably more scatter between the participating codes.  \aspect
+does, however, fall within the range of the other results, and the curve is somewhat less wiggly.
+The results for maximum topography versus time are shown in~\ref{fig:crameri-2-comparison}
+
+The precise values for topography at a given time are quite dependent on the resolution and
+timestepping parameters. Following \cite{CSG12} we investigate the convergence of the maximum 
+topography at 3 Ma as a function of CFL number and mesh resolution.  The results are shown in 
+figure~\ref{fig:crameri-benchmark-convergence}. 
+
+\begin{figure}
+  \begin{center}
+    \includegraphics[width=0.95\textwidth]{cookbooks/benchmarks/crameri/crameri_2_convergence}
+  \end{center}
+  \caption{\it Convergence for case two.  Left: Logarithm of the error with decreasing CFL number. 
+As the CFL number decreases, the error gets smaller. However, once it reaches a value of $\sim0.1$, there
+stops being much improvement in accuracy. Right: Logarithm of the error with increasing maximum mesh 
+resolution. As the resolution increases, so does the accuracy.}
+  \label{fig:crameri-benchmark-convergence}
+\end{figure}
+
+We find that at 3 Ma \aspect  converges to a maximum topography of $\sim$396 meters. 
```

Comment:
Here (and in a couple of other places) I think you need to write it as `\aspect{}` for some reasons I have forgotten, or the spacing after the command will be off. Or look at the output -- if it looks ok, then no need to worry.